helm/panda/charts/server/panda_server_config.json

```diff
@@ -17,7 +17,7 @@
         "adder_plugins": "wlcg:dataservice.AdderDummyPlugin:AdderDummyPlugin,wlcg:dataservice.AdderSimplePlugin:AdderSimplePlugin:sphenix",
         "setupper_plugins": "wlcg:dataservice.SetupperDummyPlugin:SetupperDummyPlugin", 
         "token_authType": "oidc",
-        "cache_dir": "/var/cache/pandaserver",
+        "cache_dir": "/var/log/panda/pandacache",
```

Comment:
This is inconsistent with 
https://github.com/PanDAWMS/panda-server/blob/master/templates/panda_server-httpd-FastCGI.conf.rpmnew.template#L38
so that those files cannot be downloaded. If your intention is to put those files on PV, the dir structure or httpd conf needs to be arranged accordingly.

Reply:
ok. I can create a link in the container start script.